src/components/ManageUsers/manage-users.css

```diff
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
+  display: right;
```

Comment:
there is no `display:right`. If it works it is to remove

Reply:
Already deleted